bukuserver/templates/bukuserver/bookmark_create.html

```diff
@@ -6,7 +6,7 @@
   {{ buku.set_lang() }}
   {{ buku.limit_navigation_if_popup() }}
   {{ buku.script('bookmark.js') }}
-  {{ buku.fetch_checkbox() }}
+  {{ buku.fetch_checkbox(form.fetch.data) }}
```

Comment:
Previously the initial value of this checkbox was hardcoded; now it's based on request parameters.